src/store/getters.ts

```diff
@@ -72,6 +72,11 @@ export default {
     return (state.entityType === CorpTypeCd.BC_CORPORATION)
   },
 
+  /** Is True if entity is a Firm. */
+  isFirm (state: StateIF): boolean {
+    return (state.entityType === CorpTypeCd.SOLE_PROP || state.entityType === CorpTypeCd.PARTNERSHIP)
+  },
```

Comment:
1 getter for now as i haven't seen anything differentiating the two with regards to this UI. Can be split up when/if required.